### ISSUE TITLE
Chinese text will not display in Portal 2.

### ISSUE THREAD

**Issue by Kakurady** [ValveSoftware/portal2#200]
When Portal 2 is set to display in Chinese, no Chinese text is actually rendered. That includes text in the menu and closed captions. The developer console displays "m_face->glyph->bitmap.width is 0 for ch:36827 Nimbus Sans L". English characters still render.

Happens to Simplified Chinese, Traditional Chinese, Japanese and Thai. I'm using Ubuntu 14.04.

To reproduce: 
Right-click Portal 2 in the Steam library, select Properties, select the Language tab, select Simplified Chinese. Start Portal 2.

Expected results: 
Chinese text are readable.

Actual results:
Chinese text is not displayed.
![2014-04-26_00004](https://cloud.githubusercontent.com/assets/138435/2808340/67e2de28-cd24-11e3-8150-d415c0ad8d5c.jpg)
![2014-04-26_00002](https://cloud.githubusercontent.com/assets/138435/2808341/6a8ee252-cd24-11e3-8f67-02048a70dcaa.jpg)

**Comment by ItsMeOnly** [comment]
Do you have Asian TTF fonts installed? Your URW Nimbus Sans doesn't seem to have any Chinese glyphs compiled in, maybe it's a substitute font.

**Comment by Kakurady** [comment]
Yes, I have fonts-droid and fonts-wqy-microhei installed; but Portal 2 is not using them.

**Comment by ItsMeOnly** [comment]
Maybe that's the problem: set up a font substitution, because it looks for "Nimbus Sans"

**Comment by CaulyKan** [comment]
create a file at ~/.fonts.conf should work:

```
<?xml version="1.0"?>
<!DOCTYPE fontconfig SYSTEM "fonts.dtd">
<fontconfig>
  <match target="pattern">
    <test name="family" qual="any" >
      <string>Nimbus Sans L</string>
    </test>
    <edit name="family" mode="assign" binding="same">
      <string>WenQuanYi Micro Hei</string>
    </edit>
  </match>

</fontconfig>
```

**Comment by mousemuffins** [comment]
How do I create a file at ~/.fonts.conf ? I am having this same issue and have not found a solution either. Where is the ~ ?

**Comment by jedenastka** [comment]
`~` is your home folder.
